FaceDetection/CMakeLists.txt

```diff
@@ -1,17 +1,23 @@
-cmake_minimum_required(VERSION 2.8.4)
+cmake_minimum_required(VERSION 3.5.1)
```

Comment:
CMake 3.1.0 or newer is OK for everything except for `OpenMP_FOUND`. So maybe we can use `OPENMP_FOUND` and `cmake_minimum_required(VERSION 3.1.0)`?